src/js/stores/PositionStore.js

```diff
@@ -0,0 +1,80 @@
+import service from '../utils/service';
+import { createStore } from '../utils/createStore';
+import { shallowClone } from '../utils/object-utils';
+
+const AppDispatcher = require('../dispatcher/AppDispatcher');
+const PositionConstants = require('../constants/PositionConstants');
+const PositionActions = require('../actions/PositionActions');
+
+var _position_store = {}; // All positions that have been fetched (by we_vote_ids)
+
+function printErr (err) {
+  console.error(err);
+}
+
+const POSITION_CHANGE_EVENT = 'POSITION_CHANGE_EVENT';
+
+const PositionAPIWorker = {
+
+  positionRetrieve: function (we_vote_id, success) {
+    return service.get({
+      endpoint: 'positionRetrieve',
+      query: {
+         position_we_vote_id: we_vote_id,
+      },
+      success
+    });
+  }
+
+};
+
+const PositionStore = createStore({
+
+  emitChange: function () {
```

Comment:
we can remove line **33** to **45** because `createStore()` function extends object with `EventEmitter.prototype` and these 3 functions.